src/apps/something-similar/SomethingSimilar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useEffect, useState } from "react";
+import { getWorkIdFromQueryParam } from "../../core/utils/helpers/url";
+import { FaustId } from "../../core/utils/types/ids";
+import SomethingSimilarList from "./SomethingSimilarList";
+
+const SomethingSimilar: FC = () => {
+  const [loanForRecommender, setLoanForRecommender] = useState<string | null>(
+    null
+  );
+
+  useEffect(() => {
+    setLoanForRecommender(getWorkIdFromQueryParam());
```

Comment:
Please see previous comment about feeding the app with a work id instead of fetching it from the url